templates/themes/light/style.html.jinja

```diff
@@ -0,0 +1,72 @@
+<style>
+
+    body {
+        background-color: #FAFAFA;
+        font-family: "Arial", sans-serif;
+    }
+
+    div#container {
+        margin: auto;
+        max-width: 640px;
+        padding: 10px;
+        text-align: center;
+    }
+
+    div.post {
+        margin-top: 20px;
+    }
+
+    h1 {
+        color: black;
+    }
+
+    h2 {
+        color: black;
+    }
+
+    a.home-link {
+        color: black;
+    }
+
+    a.original-link {
+        color: black;
+    }
+
+    span.link-divider {
+        color: black;
+    }
+
+    span#render-description {
+        color: #333;
+    }
+
+    span#scorer {
+        color: #333;
+    }
+
+    span#scorer-value {
```

Comment:
The html template uses `<span class="render-info-value">`, but this (and the lines below) are defined as IDs, so the styles aren't applying.

Reply:
Ah, this got out of sync when rebasing previous changes.